tutorials/terraform/sqoop/clusters-postgresql-data-proc-and-vm.tf

```diff
@@ -0,0 +1,257 @@
+# Infrastructure for Yandex Cloud Managed Service for PostgreSQL cluster.
+#
+# RU: https://cloud.yandex.ru/docs/managed-postgresql/tutorials/sqoop
+#
+# Set the configuration of the Managed Service for PostgreSQL cluster, Managed Service for Data Proc cluster and Virtual machine:
+locals {
+  folder_id           = ""      # Your folder ID.
+  network_id          = ""      # Network ID for Managed Service for PostgreSQL cluster, Data Proc cluster and VM.
+  subnet_id           = ""      # Subnet ID (enable NAT for this subnet).
+  storage_sa_id       = ""      # Service account ID for creating a bucket in Object Storage.
+  data_proc_sa        = ""      # Set a Data Proc service account name. It must be unique in folder.
+  pg_cluster_version  = "14"    # Set the PostgreSQL version. See full list of supported versions here: https://cloud.yandex.com/en/docs/managed-postgresql/.
```

Comment:
Set the PostgreSQL version. See the complete list of supported versions in https://cloud.yandex.com/en/docs/managed-postgresql/.